manticore/core/workspace.py

```diff
@@ -59,6 +59,27 @@ class Store(object):
     Implement either save_value/load_value in subclasses, or save_stream/load_stream, or both.
     """
 
+    @classmethod
+    def fromdescriptor(cls, desc):
+	"""
+	Create a :class:`~manticore.core.workspace.Store` instance depending on the descriptor.
+
+	Valid descriptors:
+	  * fs:<path>
+	  * redis:<hostname>:<port>
+	  * mem:
+
+	:param str desc: Store descriptor
+	:return: Store instance
+	"""
+        type_, uri = ('fs', None) if desc is None else desc.split(':', 1)
+        for subclass in cls.__subclasses__():
+            if not hasattr(subclass, 'store_type'):
```

Comment:
i'm wondering if we need an explicit check here. accessing `subclass.store_type` in the next line will error and implicitly communicate that same message to a Store subclass implementer that forgot it?